### DISCUSSION TITLE
VBET Runs for November 2022

### DISCUSSION THREAD

**Discussion by MattReimer** [Riverscapes/riverscapes-tools#637] [VBET] [answered]
FYI @joewheaton @philipbaileynar @KellyMWhitehead @shelbysawyer @lauren-herbine 

We've got VBET into a stable-ish and performant-ish state so now it's time to prove us wrong and run a few HUC8s and take a look:

_**NB:** This table is copied and pasted from a google sheet: https://docs.google.com/spreadsheets/d/1oCVs7Pa2zokAMICNL2X97fqVOsXlx12NKb5KnwRbeFo/edit#gid=1423074970_

HUC8 | Name |   |   | notes
-- | -- | -- | -- | --
05010003 | Middle Allegheny-Tionesta | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/b73a45d2-77ea-4071-89f2-76838ca5fa67) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/b73a45d2-77ea-4071-89f2-76838ca5fa67) |  
05020004 | Cheat | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/02b3454e-4130-4881-b872-1cbee9aff852) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/02b3454e-4130-4881-b872-1cbee9aff852) |  
05040003 | Walhonding | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/dcc50b85-960c-4173-a9e6-7d61f7a439c8) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/dcc50b85-960c-4173-a9e6-7d61f7a439c8) |  
05140201 | Lower Ohio-Little Pigeon | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/cc547488-4436-413a-8439-4978dc495563) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/cc547488-4436-413a-8439-4978dc495563) |  
06010202 | Upper Little Tennessee | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/9bcda430-7d33-4d59-b454-789281468ec5) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/9bcda430-7d33-4d59-b454-789281468ec5) |  
06010207 | Lower Clinch | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/a5479be8-aa19-414a-b57d-6ea2ce124f7a) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/a5479be8-aa19-414a-b57d-6ea2ce124f7a) |  
07020005 | Chippewa | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/49936480-af64-4f7a-96aa-fdbfa7e1e393) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/49936480-af64-4f7a-96aa-fdbfa7e1e393) |  
07050002 | Flambeau | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/7ec67eee-3cee-4f10-ad83-e70547e82ded) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/7ec67eee-3cee-4f10-ad83-e70547e82ded) |  
10030201 | Two Medicine | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/c0f7521c-97b8-40c1-bd25-ca5c53436738) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/c0f7521c-97b8-40c1-bd25-ca5c53436738) |  
10050015 | Rock | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/2bd97f8b-b480-4b44-b93d-9179d66166f9) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/2bd97f8b-b480-4b44-b93d-9179d66166f9) | _possible Canadian border problem_ #639
10190002 | Upper South Platte | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/11df3437-4e63-4bc6-a51e-1279c57f66fc) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/11df3437-4e63-4bc6-a51e-1279c57f66fc) |  
10190003 | Middle South Platte-Cherry Creek | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/5cdcbee8-063a-4f9f-8a91-59fe56d706c8) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/5cdcbee8-063a-4f9f-8a91-59fe56d706c8) |  
10190004 | Clear | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/f1710d88-ff50-4f72-b77a-69547a145ae0) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/f1710d88-ff50-4f72-b77a-69547a145ae0) |  
10190005 | St. Vrain | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/3bfff8fb-1e81-4ee3-a473-4037a8ec794c) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/3bfff8fb-1e81-4ee3-a473-4037a8ec794c) |  
10190010 | Kiowa | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/76ffa7d8-2926-40c9-9b3b-1723ed1bba52) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/76ffa7d8-2926-40c9-9b3b-1723ed1bba52) |  
16010201 | Bear Lake | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/eece7d9d-97be-4c39-842c-9747ee26b56b) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/eece7d9d-97be-4c39-842c-9747ee26b56b) |  
16010202 | Middle Bear | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/d8485cc8-db2c-4b59-b410-a92f26a375dc) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/d8485cc8-db2c-4b59-b410-a92f26a375dc) |  
16010203 | Little Bear-Logan | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/07bb6d61-f113-48ab-bcec-178de9db3d12) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/07bb6d61-f113-48ab-bcec-178de9db3d12) |  
17060304 | Middle Fork Clearwater | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/8c0b7099-1c55-4c8c-bb94-d98e7a29087f) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/8c0b7099-1c55-4c8c-bb94-d98e7a29087f) |  
1601020201 | Middle Bear (HUC10) | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/8239d22d-8d64-4c91-b184-f4f28458bcfb) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/8239d22d-8d64-4c91-b184-f4f28458bcfb) |  
1601020204 | Middle Bear (HUC10) | [Warehouse](https://northarrowresearchlabs.github.io/riverscapes-staging/#/Anabranch/2518e88e-0d6c-4489-b303-d0d10ea2e35e) | [RV](https://northarrowresearchlabs.github.io/webrave-staging/#/Anabranch/2518e88e-0d6c-4489-b303-d0d10ea2e35e) |  

All of these are running in cybercastor. I'll populate the links above as they become available.

### What we're looking for here:

1. Download (or view in WebRAVE) the hucs.
2. Add any questions to this conversation.
3. We will create issues if anything is seriously wrong.

### Notes:

- There may be some businesslogic problems, especially on vectors you see in RV. Best to download and use QRave or ArcRave so you can view the layers properly.

**Comment by joewheaton** [comment, marked as answer]
Well these all got run (Thank you) and are now superseded by #651.
